waterlog/src/Components/MonthlyUsage.js

```diff
@@ -37,16 +35,15 @@ const MonthlyUsageComponent = (props) => {
         display: true,
         scaleLabel: {
           display: true,
-          labelString: 'Liters'
+          labelString: 'liters'
```

Comment:
If only these were in constants then you had to only change it once

Reply:
I think make use of contants here

Reply:
Thanks for the input I will consider doing that.